CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- B2B impersonation not loading profile page
```

Comment:
```suggestion
- B2B impersonation not loading profile page

```

Reply:
This entry is describing the effect your fix had on the `my-account` interface, and not quite what was changed in this app (store-graphql), so I would go for something along the lines of:

```suggestion
- `withCurrentProfile` directive was not taking into account `session.public.impersonate` field from the current session when computing the value for the `userType` field.

```